bots/images/scripts/fedora.setup

```diff
@@ -17,7 +17,9 @@ fi
 # depending on how it was compiled and which plugins are installed,
 # and will just silently not do it.  So we do it here.
 #
-xfs_growfs /
+if [ "$IMAGE" != fedora-31 ]; then
+    xfs_growfs /
+fi
```

Comment:
This seems awfully specific -- what's the difference here, F31 switched to something else than XFS? Or can this resizing be dropped for other images as well?

Reply:
It's not available (yet) on F31. xfsprogs hasn't been released yet to f31 either.

Reply:
At least on my slightly older fedora-31 image I see

    /dev/mapper/fedora-root  9.6G  4.8G  4.4G  52% /

Which seems fine. So perhaps that whole thing is obsolete for the other images, too. We can clean these up in a follow-up, so this is fine for now.